src/snowflake/cli/api/utils/rendering.py

```diff
@@ -88,22 +94,151 @@ def jinja_render_from_file(
         print(rendered_result)
 
 
-class _AttrGetter:
+def _is_key_in_environ(key: str):
+    return key.upper() in os.environ or key in os.environ
```

Comment:
```suggestion
    return key in os.environ
```
See below for rationale.